crates/bench/benches/db.rs

```diff
@@ -12,54 +12,89 @@ fn build_group<'a>(c: &'a mut Criterion, named: &str, run: Runs) -> BenchmarkGro
     group
 }
 
-fn bench_insert_tx_per_row(c: &mut Criterion) {
+fn bench_insert_tx_per_row(c: &mut Criterion, fsync: bool) {
```

Comment:
I'm wondering if we want to plan ahead a tiny bit and make this something more like


```rust
enum FsyncPolicy {
    Never,
    EveryTx,
}
```

so we can later add an `EveryNMilliseconds(usize)` version etc